core/types/receipt.go

```diff
@@ -609,6 +553,77 @@ func convertLogsForEncoding(logs []*Log) []*LogForStorage {
 	return result
 }
 
+func decodeStoredQuorumReceiptExtraDataV1(r *QuorumReceiptExtraData, blob []byte) error {
```

Comment:
Just `decodeQuorumReceiptExtraDataV1` or
```suggestion
func decodeAndStoreQuorumReceiptExtraDataV1(r *QuorumReceiptExtraData, blob []byte) error {
```
?

Reply:
It is derived from the name of the structure `storedQuorumReceiptExtraDataV1RLP` with the `decode` prefix... I'd prefer to keep it that way. 
There is no `QuorumReceiptExtraDataVxyz` - we don't have versions for that. We only have versions for the stored structure.